### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by 1ec5**
Removed the simpler, pre-v1.0 installation instructions. Updated the Xcode project, CocoaPods podspecs, and readme for v1.0.0. Copyedited the changelog.

Tasks:

* [x] Remove pre-v1.0 installation instructions
* [x] Copyedit the changelog
* [x] Upgrade Mapbox dependencies to v1.0.0
   * [x] Update changelog to reflect the new dependency versions
* [x] Increase the major version number

Blocked by mapbox/turf-swift#116, mapbox/mapbox-speech-swift#39, and mapbox/mapbox-directions-swift#463.

/cc @mapbox/navigation-ios

### CODE REVIEW

**Review by Udumft — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
...
+* By default, usage of Mapbox APIs is now [billed](https://www.mapbox.com/pricing/) together based on [monthly active users](https://docs.mapbox.com/help/glossary/monthly-active-users/) rather than individually by HTTP request. If you prefer to still use request-based billing, set the `MBXNavigationBillingMethod` key in Info.plist to `request`. ([#2405](https://github.com/mapbox/mapbox-navigation-ios/pull/2405))
 * Added Greek and Turkish localizations. ([#2385](https://github.com/mapbox/mapbox-navigation-ios/pull/2385), [#2475](https://github.com/mapbox/mapbox-navigation-ios/pull/2475), [valhalla/valhalla#2438](https://github.com/valhalla/valhalla/pull/2438))
 * Upgraded to [MapboxDirections v1.0.0-rc.2](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v1.0.0-rc.2). ([#2619](https://github.com/mapbox/mapbox-navigation-ios/pull/2619)) 
 * Upgraded to [MapboxSpeech v0.3.1](https://github.com/mapbox/mapbox-speech-swift/releases/tag/v0.3.1). ([#2543](https://github.com/mapbox/mapbox-navigation-ios/pull/2543))
```
Comment (1ec5) [inline CHANGELOG.md:24]: We’ll need to upgrade both dependencies and Turf to v1.0.0 once it comes out, not only in here in the changelog but also in the Cartfile and podspecs.

**Review by MaximAlien — COMMENTED** [review]
Probably also good idea to run `$ pod update` for `CocoaPodsTest` if you haven't done that already.

**Comment by 1ec5** [comment]
> Probably also good idea to run `$ pod update` for `CocoaPodsTest` if you haven't done that already.

Good point, I ran that in #2649 but not here. That said, I think it’s still the case that the CocoaPods installation test fails for the release PR in particular because the release hasn’t been pushed to CocoaPods trunk yet.

**Comment by 1ec5** [comment]
>> Probably also good idea to run `$ pod update` for `CocoaPodsTest` if you haven't done that already.
> 
> We should add that to the release script too.

#2650 updates the release script in time for v1.0.0-rc.5. We’ll need to bump the build number in this PR if that RC lands first.

**Comment by 1ec5** [comment]
codecov thinks our code coverage has [gone down](https://codecov.io/gh/mapbox/mapbox-navigation-ios/compare/71e384a195a4fcb9a59674c09968872b7a78cf56...2f3c73cd677168cf32e7710f8c0be090672a5fe9) by −44.64% to 13.39%. Obviously that’s wrong and it isn’t the first time, so I’m going to merge anyways.

**Comment by 1ec5** [comment]
Cherry-picked into the release-v1.0 branch as 27e8aa9ebd4b518bdf8c755c7f37e4e55b0e2de6.

**Comment by 1ec5** [comment]
We had to abort and roll back this release because of a linker error when pushing to CocoaPods trunk:

```sh
pod trunk push MapboxCoreNavigation.podspec --skip-import-validation --allow-warnings
```

```
 -> MapboxCoreNavigation (1.0.0)
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Building targets in parallel
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | [iOS] xcodebuild:  note: Planning build
    - NOTE  | [iOS] xcodebuild:  note: Constructing build description
    - NOTE  | [iOS] xcodebuild:  warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99. (in target 'Polyline' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99. (in target 'MapboxMobileEvents' from project 'Pods')
    - NOTE  | [iOS] xcodebuild:  ld: warning: ignoring file MapboxCommon/MapboxCommon.xcframework/ios-armv7_arm64/MapboxCommon.framework/MapboxCommon, missing required architecture x86_64 in file MapboxCommon/MapboxCommon.xcframework/ios-armv7_arm64/MapboxCommon.framework/MapboxCommon (2 slices)
    - NOTE  | [iOS] xcodebuild:  ld: warning: ignoring file MapboxCommon/MapboxCommon.xcframework/ios-armv7_arm64/MapboxCommon.framework/MapboxCommon, missing required architecture i386 in file MapboxCommon/MapboxCommon.xcframework/ios-armv7_arm64/MapboxCommon.framework/MapboxCommon (2 slices)
    - NOTE  | xcodebuild:  clang: error: linker command failed with exit code 1 (use -v to see invocation)
    - NOTE  | [iOS] xcodebuild:  warning: [CP] Unable to find matching .xcframework slice in ' true ios-armv7_arm64/MapboxCommon.framework ios-x86_64-simulator/MapboxCommon.framework ios-x86_64-maccatalyst/MapboxCommon.framework' for the current build architectures (arm64 i386 x86_64).
    - NOTE  | [iOS] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
[!] The spec did not pass validation, due to 1 error.
```

```
    Ld /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/x86_64/Binary/MapboxCoreNavigation normal x86_64 (in target 'MapboxCoreNavigation' from project 'Pods')
        cd /var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods
        /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -target x86_64-apple-ios10.0-simulator -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk -L/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxDirections -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxMobileEvents -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/Polyline -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/Turf -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxAccounts -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxCommon -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxNavigationNative -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/XCFrameworkIntermediates/MapboxCommon -filelist /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/x86_64/MapboxCoreNavigation.LinkFileList -install_name @rpath/MapboxCoreNavigation.framework/MapboxCoreNavigation -Xlinker -rpath -Xlinker /usr/lib/swift -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/x86_64/MapboxCoreNavigation_lto.o -Xlinker -objc_abi_version -Xlinker 2 -fobjc-arc -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -L/usr/lib/swift -Xlinker -add_ast_path -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/x86_64/MapboxCoreNavigation.swiftmodule -framework MapboxAccounts -framework MapboxCommon -framework MapboxNavigationNative -framework Foundation -framework MapboxDirections -framework MapboxMobileEvents -framework Turf -compatibility_version 1 -current_version 1 -Xlinker -dependency_info -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/x86_64/MapboxCoreNavigation_dependency_info.dat -o /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/x86_64/Binary/MapboxCoreNavigation
    ld: warning: directory not found for option '-F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/XCFrameworkIntermediates/MapboxCommon'
    ld: framework not found MapboxCommon
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    
    Ld /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/i386/Binary/MapboxCoreNavigation normal i386 (in target 'MapboxCoreNavigation' from project 'Pods')
        cd /var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods
        /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -target i386-apple-ios10.0-simulator -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk -L/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxDirections -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxMobileEvents -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/Polyline -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/Turf -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxAccounts -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxCommon -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxNavigationNative -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/XCFrameworkIntermediates/MapboxCommon -filelist /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/i386/MapboxCoreNavigation.LinkFileList -install_name @rpath/MapboxCoreNavigation.framework/MapboxCoreNavigation -Xlinker -rpath -Xlinker /usr/lib/swift -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/i386/MapboxCoreNavigation_lto.o -Xlinker -objc_abi_version -Xlinker 2 -fobjc-arc -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -L/usr/lib/swift -Xlinker -add_ast_path -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/i386/MapboxCoreNavigation.swiftmodule -framework MapboxAccounts -framework MapboxCommon -framework MapboxNavigationNative -framework Foundation -framework MapboxDirections -framework MapboxMobileEvents -framework Turf -compatibility_version 1 -current_version 1 -Xlinker -dependency_info -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/i386/MapboxCoreNavigation_dependency_info.dat -o /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/i386/Binary/MapboxCoreNavigation
    ld: warning: directory not found for option '-F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/XCFrameworkIntermediates/MapboxCommon'
    ld: framework not found MapboxCommon
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    
    Ld /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/arm64/Binary/MapboxCoreNavigation normal arm64 (in target 'MapboxCoreNavigation' from project 'Pods')
        cd /var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods
        /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -target arm64-apple-ios10.0-simulator -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk -L/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxDirections -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxMobileEvents -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/Polyline -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/Turf -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxAccounts -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxCommon -F/var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods/MapboxNavigationNative -F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/XCFrameworkIntermediates/MapboxCommon -filelist /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/arm64/MapboxCoreNavigation.LinkFileList -install_name @rpath/MapboxCoreNavigation.framework/MapboxCoreNavigation -Xlinker -rpath -Xlinker /usr/lib/swift -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/arm64/MapboxCoreNavigation_lto.o -Xlinker -objc_abi_version -Xlinker 2 -fobjc-arc -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -L/usr/lib/swift -Xlinker -add_ast_path -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/arm64/MapboxCoreNavigation.swiftmodule -framework MapboxAccounts -framework MapboxCommon -framework MapboxNavigationNative -framework Foundation -framework MapboxDirections -framework MapboxMobileEvents -framework Turf -compatibility_version 1 -current_version 1 -Xlinker -dependency_info -Xlinker /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/arm64/MapboxCoreNavigation_dependency_info.dat -o /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/arm64/Binary/MapboxCoreNavigation
    ld: warning: directory not found for option '-F/Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/XCFrameworkIntermediates/MapboxCommon'
    ld: framework not found MapboxCommon
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    
    RegisterExecutionPolicyException /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation/MapboxCoreNavigation.framework (in target 'MapboxCoreNavigation' from project 'Pods')
        cd /var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods
        builtin-RegisterExecutionPolicyException /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation/MapboxCoreNavigation.framework
    
    Touch /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation/MapboxCoreNavigation.framework (in target 'MapboxCoreNavigation' from project 'Pods')
        cd /var/folders/cl/8cgpg37n5cl78pjkbpgml94w0000gn/T/CocoaPods-Lint-20200924-30942-14b8v3f-MapboxCoreNavigation/Pods
        /usr/bin/touch -c /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Products/Release-iphonesimulator/MapboxCoreNavigation/MapboxCoreNavigation.framework
    
    ** BUILD FAILED **
    
    
    The following build commands failed:
    	Ld /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/x86_64/Binary/MapboxCoreNavigation normal x86_64
    	Ld /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/i386/Binary/MapboxCoreNavigation normal i386
    	Ld /Users/mxn/Library/Developer/Xcode/DerivedData/App-drpkpdizctsxuqdysasgarabgcqb/Build/Intermediates.noindex/Pods.build/Release-iphonesimulator/MapboxCoreNavigation.build/Objects-normal/arm64/Binary/MapboxCoreNavigation normal arm64
    (3 failures)
```

**Comment by zugaldia** [comment]
The team agreed to continue the process on Monday.

**Comment by 1ec5** [comment]
#2662 should fix the linker errors when pushing to CocoaPods trunk, allowing us to retry the release later this week.

**Comment by 1ec5** [comment]
v1.0.0 is now available [on GitHub](https://github.com/mapbox/mapbox-navigation-ios/releases/tag/v1.0.0) for Carthage users and on CocoaPods trunk (CocoaPods/Specs@e19b50fccfefcd89eddd80822fce3a0c03853932, CocoaPods/Specs@3c76658855cd8a748e49c15c745a738d6d0cdcc2).

```
--------------------------------------------------------------------------------
 🎉  Congrats

 🚀  MapboxCoreNavigation (1.0.0) successfully published
 📅  October 1st, 17:13
 🌎  https://cocoapods.org/pods/MapboxCoreNavigation
 👍  Tell your friends!
--------------------------------------------------------------------------------
```

```
--------------------------------------------------------------------------------
 🎉  Congrats

 🚀  MapboxNavigation (1.0.0) successfully published
 📅  October 1st, 17:45
 🌎  https://cocoapods.org/pods/MapboxNavigation
 👍  Tell your friends!
--------------------------------------------------------------------------------
```

**Pull request merged**